src/main/java/com/example/carservice/controller/CarController.java

```diff
@@ -14,12 +13,17 @@ public class CarController {
 
     private final CarService service;
 
-    public CarController(CarService service) {
+    public CarController(CarService service, CarRepository carRepository) {
```

Comment:
Why are you using the 'carRepository' again? It seems that we decided to use just a 'repository'.